src/main/java/org/htwkvisu/gui/MapCanvas.java

```diff
@@ -28,14 +31,13 @@
     private static final MouseButton MOUSEBUTTON_SELECT = MouseButton.PRIMARY;
     private static final int SELECTION_MAX_PX_TOLERANCE = 10;
 
-    private static final double MIN_SCORING_VALUE = 0;
-    private static final double MAX_SCORING_VALUE = 1000; //TODO MAX?
-
     // scores
     private Point2D mapCenter = new Point2D(51.343479, 12.387772);
     private LinkedList<IMapDrawable> drawables = new LinkedList<>();
-    private int samplingPixelDensity = 20;
-    private ScoringCalculator calculator;
+
+    private int samplingPixelDensity = 500;
```

Comment:
Kann in einer externen Klasse gehalten werden, die sich um das NumericField zum einen kümmert, die DefaultValues mit ihren Grenzen kennt und letztendlich dem MapCanvas den richtigen ScoringValue gibt, wenn er fragt. Der Grund für eine solche Extraktion wäre hier: Zeilen reduzieren.